core/src/themes/native/test/colors/theme.e2e.ts

```diff
@@ -31,7 +31,7 @@ const styleTestHelpers = `
       background: rgba(var(--ion-color-base-rgb), 0.16);
     }
     .ion-color {
-      color: var(--ion-color-base);
+      color: var(--ion-color-foreground);
```

Comment:
Should we also update the native colors page? Like this [line](https://github.com/ionic-team/ionic-framework/blob/ad885c1c222dcfa704a4548ed84ba91d01145afa/core/src/themes/native/test/colors/index.html#L49)?

Reply:
We can just as another way to make sure the fallback is working. Updated: https://github.com/ionic-team/ionic-framework/pull/30306/commits/2a92d81ed9f40aeb5b3af484fdbcf0ec584eb540

Reply:
There's a few other instances that should be changed in this file. I didn't link them so you wouldn't get spammed with links.

Reply:
Where? I only see `.ion-color` used once.

Reply:
I don't know actually how we will be using those yet, since the `foreground` was created for text-only color components without a background and those are all using "subtle" backgrounds. Right now we don't have native using a different `foreground` at all, so changing them won't do anything. I can update it though in case it catches something in the future.

Reply:
Updated https://github.com/ionic-team/ionic-framework/pull/30306/commits/7174092fdb41e21fa62219754b46a4232559afbb